enterprise/templates/enterprise/admin/setup_auth_org_id.html

```diff
@@ -0,0 +1,52 @@
+{% extends "admin/base_site.html" %}
+{% load i18n static admin_urls %}
+
+{% block breadcrumbs %}
+<div class="breadcrumbs">
+  <a href="{% url 'admin:index' %}">{% trans "Home" %}</a>
+  &rsaquo; <a href="{% url 'admin:app_list' app_label=opts.app_label %}">{{ opts.app_config.verbose_name }}</a>
+  &rsaquo; {% if has_change_permission %}
+    <a href="{% url opts|admin_urlname:'changelist' %}">{{ opts.verbose_name_plural|capfirst }}</a>
+  {% else %}
+    {{ opts.verbose_name_plural|capfirst }}
+  {% endif %}
+  &rsaquo; {% if has_change_permission %}
+    <a href="{% url opts|admin_urlname:'change' enterprise_customer.uuid %}">
+      {{ enterprise_customer|truncatewords:"18" }}
+    </a>
+  {% else %}
+    {{ enterprise_customer|capfirst }}
+  {% endif %}
+  &rsaquo;
+  {% trans "Setup auth org ID" %}
```

Comment:
I don't know where to put this translation, do I need to run a make command or something?

Reply:
You can run `make extract_translations` I think? https://github.com/openedx/edx-platform/blob/2209549e68776f5db3c77a571bba115918fcf3f7/Makefile#L40-L53
That said, you're making changes to an internal tool, so I wouldn't let this be a blocker.